OvmfPkg/OvmfPkgX64.dsc

```diff
@@ -707,6 +747,21 @@
 !endif
   }
 
+### SETUP_PASSWORD
+
+!if $(SECURE_BOOT_ENABLE) == TRUE
+  SecurityPkg/VariableAuthenticated/SecureBootConfigDxe/SecureBootConfigDxe.inf

```

Comment:
The `SecureBootConfigDxe` is already present in the file, please remove all duplicates that you added

Reply:
Sure, working on it right away.